end-to-end-test/shared/specUtils.js

```diff
@@ -699,6 +699,42 @@ function selectClinicalTabPlotType(type) {
     ).click();
 }
 
+function clickColumnSelectionButton(patientTable) {
+    $(`[data-test=${patientTable}]`)
+        .$('button*=Columns')
+        .click();
+}
+
+function selectColumn(namespaceColumn1) {
+    $(`[data-id="${namespaceColumn1}"]`).click();
+}
+
+waitForMutationTable = () => {
+    $('[data-test=LazyMobXTable]').waitForDisplayed();
+};
+
+waitForTable = table => {
```

Comment:
this can be accomplished using ```getElementByTestHandle```
we need to start using that in order to hide the whole test-handle thing.

Reply:
and again, no reason to use a helper here and clutter the specUtils.

Reply:
Fixed in [45e5ee3](https://github.com/cBioPortal/cbioportal-frontend/pull/4424/commits/45e5ee3ba491ee838276c3807739a92488f62ed1)